### PR TITLE
Issue labeling guide (suggestions welcomed)

### PR DESCRIPTION
**Pull request by kadel**

### CODE REVIEW

File: docs/issue-labeling.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+# Issue labeling guide
+
+
+## size/*
+size/* labels are for estimating size.
+It should be estimation how complicated and time consuming solving this is going to be.
+
+S - simple chage, just few lines (no more than 1 day of work)
```
Comment (pradeepto) [inline docs/issue-labeling.md]: s/chage/change

File: docs/issue-labeling.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+M - a little bit more change but still fairly strait forward
```
Comment (pradeepto) [inline docs/issue-labeling.md]: M - considerable change but fair straightforward problem statement

File: docs/issue-labeling.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+L - a bit more complicated to solve, maybe requiring small refactoring of existing code
+XL - complex change, new big feature, requiring big refactoring
```
Comment (pradeepto) [inline docs/issue-labeling.md]: XL - complex change, new big feature, requiring big refactoring, perhaps is an epic and should be broken into smaller tasks.

**Comment by surajssd** [comment]
@kadel have you got some time to address the points from @pradeepto ?

**Comment by kadel** [comment]
I think that current version is a good start. 
We can improve it as it goes and needed.

**Comment by cdrage** [comment]
@tkral Could this be in `development.md` instead? Since this is "meta" and related more to development. Having a separate doc would end up being in the side-bar of the site.

**Comment by kadel** [comment]
> @tkral Could this be in development.md instead? Since this is "meta" and related more to development. Having a separate doc would end up being in the side-bar of the site.

good point. I will move it to development.md

**Comment by kadel** [comment]
moved to development.md

**Pull request merged**